server/models/Wallet.js

```diff
@@ -81,7 +81,10 @@ class Wallet{
    * Get all the trust relationships I have requested
    */
   async getTrustRelationshipsRequested(){
-    return await this.trustRepository.getByOriginatorId(this._id);
+    const result = await this.getTrustRelationships();
+    return result.filter(trustRelationship => {
```

Comment:
this is super pretty :)

Reply:
Thanks, I feel good too when I find in this way, we are moving the logic from DB/SQL to JS code, it might bring some cost in terms of performance, that's the backside, but it brings more re-usable code and more concentrated code.